File: quests/qeynos/Barthal.pl
my $count = 0;
sub EVENT_SPAWN {
	quest::settimer("daynight",1);
}

sub EVENT_TIMER {
	$count++;
	if (($count == 1) && ($zonetime > 599 || $zonetime < 1800)) {
		quest::start(61);
	}
	if (($count == 1) && ($zonetime < 600 || $zonetime > 1799)) {
		quest::stop();
		quest::moveto(-276,-32,3.75,50);
	}	
	if ($count ==2 ) {
	$count = 0;
	quest::settimer("daynight",28);
	}
}

sub EVENT_SIGNAL {
  if($signal == 3) {
    quest::say("Heh.. Why don't you try Freeport.. or Erudin.. ANYwhere but here!");
	}
}

sub EVENT_SAY { 
if($text=~/Hail/i){
quest::say("Hail, $name! We've got the best prices in town..  No need to shop around..  Heh, heh. I think I'll make that my new slogan!"); }
}
#END of FILE Zone:qeynos  ID:1123 -- Barthal 

File: quests/qeynos/Barthal.lua
local count = 0;

function event_spawn(e)
	eq.set_timer("1",1000);
end

function event_timer(e)
	count = count + 1;
	local ztime = eq.get_zone_time();
	if((count == 1) and (ztime.zone_time > 599 and ztime.zone_time < 1800)) then
		eq.start(61);
	elseif((count == 1) and (ztime.zone_time < 600 or ztime.zone_time > 1799)) then
		eq.stop();
		eq.move_to(-276,-32,3.75,50);
	elseif(count == 2) then
		count = 0;
		eq.stop_timer("1")
		eq.set_timer("1",28000);
	end
end

function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Hail, $name! We've got the best prices in town..  No need to shop around..  Heh, heh. I think I'll make that my new slogan!");
	end
end

function event_signal(e)
	if(e.signal == 3) then
		e.self:Say("Heh.. Why don't you try Freeport.. or Erudin.. ANYwhere but here!");
	end
end
